kuma/urls.py

```diff
@@ -95,7 +95,7 @@
 )
 urlpatterns += [re_path("users/", include("kuma.users.urls"))]
 urlpatterns += i18n_patterns(
-    re_path(r"^payments/$", payment_views.contribute, name="payments"),
+    url(r"^payments/?$", payment_views.index, name="payments"),
```

Comment:
Ah yeah, fixed that earlier (it was breaking the build).

Reply:
Good question-- I copied a pattern I saw with `/dashboards`, since it was similar to what I needed for `/payments`: 
```
urlpatterns += i18n_patterns(
    re_path(r"dashboards/?$", dashboards_index, name="dashboards.index"),
)
urlpatterns += i18n_patterns(
    re_path(r"^dashboards/", include(dashboards_lang_urlpatterns))
)
```

Should I keep as is or do we have a preference?

Reply:
What most likely matters is precedence. On document view, site search, view profile, edit profile, we don't use a trailing slash. Consistency is often better than perfection. 
I'm aware of pros and cons for and without a trailing slash.

Reply:
Got it, removed the trailing slash so now it's `re_path(r"payments$", ...`